specs/subresourceintegrity/spec.markdown

```diff
@@ -263,9 +263,9 @@ list, and use that metadata to validate the resource (as described below in
 the "[parse metadata][parse]" and "[get the strongest metadata from
 set][get-the-strongest]" algorithms).
 
-When a hash function is determined to be insecure, user agents MUST deprecate
-and eventually remove support for integrity validation using that hash
-function.
+User agents MAY deprecate support (by blocking loads) for integrity validation using 
```

Comment:
Wouldn't a user agent which has deprecated and/or removed `sha256` just load the subresource without performing any integrity checks? It would be like the case where the only hash algorithm in the integrity metadata is an unknown one (e.g. `md5`).